modules/rest/src/main/java/com/enonic/xp/app/contentstudio/rest/resource/project/ProjectResource.java

```diff
@@ -163,11 +167,69 @@ public boolean delete( final DeleteProjectParamsJson params )
     @Path("list")
     public ProjectsJson list()
     {
-        final List<ProjectJson> projects = this.projectService.list().stream().map( this::doCreateJson ).collect( Collectors.toList() );
+        final List<ProjectJson> projects = this.projectService.list().stream().map( this::doCreateJson ).collect( toList() );
 
         return new ProjectsJson( projects );
     }
 
+    @GET
+    @Path("listWithMissing")
```

Comment:
This is not a good name. list with missing what? if this is a complete list (as opposed to `list` being limited to only projects available for the current user), then it should be called `all` or `allProjects`. Or this one should be renamed to `listAll` and the other one to `listAvailable`